notebook_sandbox/jwst_boxcar/boxcar_extraction.ipynb

```diff
@@ -0,0 +1,403 @@
+      "text/plain": [
+       "<Figure size 720x432 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Plot along cross-disperion cut showing the extraction parameters\n",
+    "fig, ax = plt.subplots(figsize=(10, 6))\n",
+    "y = np.arange(image.shape[0])\n",
+    "ax.plot(y, image[:,70], 'k-')\n",
+    "mm = np.array([ext_center, ext_center])\n",
+    "mm_y = ax.get_ylim()\n",
+    "\n",
+    "ax.plot(mm, mm_y, 'b--')\n",
+    "ax.plot(mm - ext_width/2., mm_y, 'g:')\n",
+    "ax.plot(mm + ext_width/2., mm_y, 'g:')\n",
+    "\n",
+    "ax.set_title(\"Cross-dispersion Cut at Pixel=70\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Extract"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "# define the Trace\n",
+    "trace = FlatTrace(image, ext_center)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# extract\n",
+    "boxcar = BoxcarExtract()\n",
+    "\n",
+    "boxcar.width = ext_width\n",
+    "spectrum = boxcar(image, trace)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# this one crashes with \"UnitConversionError: 'pix' and 'Angstrom' (length) are not convertible\"\n",
```

Comment:
this looks like a `Spectrum1D` quirk. the units are correctly set to `u.pixel` where this spectrum is created which is intended because wavelength calibration hasn't happened yet.